Armadillo/src/main/java/com/scribd/armadillo/encryption/SecureStorage.kt

```diff
@@ -59,13 +60,23 @@ internal class ArmadilloSecureStorage @Inject constructor(
     }
 
     override fun saveDrmDownload(context: Context, audioUrl: String, drmDownload: DrmDownload) {
-        val key = getDrmDownloadKey(audioUrl, drmDownload.drmType)
+        val alias = getDrmDownloadAlias(audioUrl, drmDownload.drmType)
         val value = Base64.encodeToString(Json.encodeToString(drmDownload).toByteArray(StandardCharsets.UTF_8), Base64.NO_WRAP)
-        drmDownloadStorage.edit().putString(key, value).apply()
+        secureDrmStorage.edit().putString(alias, value).apply()
     }
 
-    override fun getDrmDownload(context: Context, audioUrl: String, drmType: DrmType): DrmDownload? =
-        drmDownloadStorage.getString(getDrmDownloadKey(audioUrl, drmType), null)?.decodeToDrmDownload()
+    override fun getDrmDownload(context: Context, audioUrl: String, drmType: DrmType): DrmDownload? {
+        val alias = getDrmDownloadAlias(audioUrl, drmType)
+        var download = secureDrmStorage.getString(alias, null)?.decodeToDrmDownload()
+        if (download == null && drmDownloadStorage.contains(alias)) {
+            //migrate old storage to secure storage
```

Comment:
We haven't released this to anyone yet, have we? Do we need to migrate this instead of just replacing it?

Reply:
Its been here in this repo for months though.  it felt irresponsible to leave it off.

Reply:
Yeah this hasn't been released so the unsecured drm storage has never been used, we probably don't need to migrate. But I understand that this is technically a public library so we probably still need to do that

Reply:
Yeah, that's fair.